test/fixtures/bind-value/expected.jsx

```diff
@@ -2,8 +2,10 @@
     <input value={ x.value } />
     <input value={x.value} onChange={ev => x.value = ev.target.value} />
     <input type="text" value={value} onChange={ev => value = ev.target.value} />
+    <input value={x.value} onChange={ev => (x.value = ev.target.value, (() => unrelatedHandler())(ev))} />
+    <input value={x.value} onChange={ev => (x.value = ev.target.value, (ev => x.value = ev.target.value)(ev))} />
```

Comment:
I think we can detect that it's an arrow function and append the bodies together - but that's a little more tricky to get correct, so we can do that as a later optimization. It would require alpha-renaming of the arrow function arg to `ev` - not sure if Babel has a utility for this already.

I was thinking there might also be an issue with capturing free variables, but since there can only be one `onChange` listener that shouldn't the the case (it's a problem in general with concatenating function bodies, e.g. it's not correct to concat `() => {var x = 2; f(x)}` and `() => g(x)` into `() => {var x = 2; f(x); g(x);}`